Solutions/Ais.Net.Specs/Ais/Net/Specs/NmeaStreamParserByLineSpecs.feature

```diff
@@ -14,63 +14,63 @@ Feature: NmeaStreamParserSpecs
 
 Scenario: Empty file
 	Given no content
-	When I parse the content
-	Then OnComplete should have been called
-    And OnNext should have been called 0 times
+	When I parse the content by line
+	Then INmeaLineStreamProcessor.OnComplete should have been called
```

Comment:
is the formatting a bit off?

Reply:
It was, but then it was before. Fixed now.